test/gurka/test_gtfs.cc

```diff
@@ -181,8 +190,11 @@ TEST(GtfsExample, WriteGtfs) {
   feed.write_trips(path_directory);
 
   // write stop_times.txt
-  std::string stopIds[3] = {stopOneID + "_" + to_string(NodeType::kMultiUseTransitPlatform),
-                            stopTwoID, stopThreeID};
+  std::string stopIds[3] = {
+      stopOneID + "_ledge_to_the_train_bucko",
+      stopTwoID,
+      stopThreeID + "_walk_the_plank_pal",
```

Comment:
have to update the trips information to reference the proper stops, you'll see that we reference the two non generated platforms by name but the middle one is generated and will use the parent station id to look up its platform when connecting during pbf generation